audio-analysis/.gitignore.txt

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
I'm still seeing some pycache in the PR. You might have to do `**/__pycache__/` to cover everything

Reply:
@tjlam oop mb mb) Removed manually and added to .gitignore. Should be good now.